src/Formats/MsgPackExtensionTypes.h

```diff
@@ -5,7 +5,7 @@ namespace DB
 
 enum class MsgPackExtensionTypes
 {
-    UUID = 0x02,
+    UUID_ = 0x02,
```

Comment:
Maybe better to rename into `MsgPackUUID`, `UUIDType`. Or if this is not compile time error silence warning.

Reply:
It will lead to compile error -`Werror=shadow`

Reply:
We can rename it into `UUIDType`.